### ISSUE TITLE
Can't create user in taiga

### ISSUE THREAD

**Issue by spicter** [fabric8io/fabric8#5389]
when trying to register at taiga nothing happens with no warning or errors showing up.

Any ideas? 

Thx

**Comment by rawlingsj** [comment]
Yeah the taiga image we're using does have an issue adding new users, I think there's a problem with the nginx config redirecting back to the taiga frontend.  Up until now we've only been using the default username/password (admin/123123) so that we can add integration into fabric8 so it works with gogs and lets chat.

This is a valid issue that needs to be fixed.

**Comment by spicter** [comment]
Thx! At least I now know the default credentials :-))

**Comment by rawlingsj** [comment]
@spicter good call :)

I've added the default username and password to the description of the template when it's run from the UI.
